### DISCUSSION TITLE
Private storage

### DISCUSSION THREAD

**Discussion by dkoci** [quay/quay#987] [Q&A] [answered]
Hi. Is it possible to use minio as private object storage? Any other private object storage?

**Comment by BillDett** [comment, marked as answer]
Sure- anything S3 compatible ought to work.

**Reply by dkoci** [reply]
I cant create a connection directly to our private s3 storage. It is working only through noobaa. Is it possible to connect quay directly to private s3 storage without noobaa? Endpoint is xxxxxx:9021
